src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -139,6 +139,44 @@ public void given_initiated_tasks_with_delayTimer_toFuture_with_followup_overdue
         assertTaskDeleteReason(task1Id, "deleted");
     }
 
```

Comment:
Can we add the test to cover AC for **makeAnApplication** event occurring and then warning message being received is not covered